### PR TITLE
Added notebook to make plot of decadal trends and plot

### PR DESCRIPTION
**Pull request by piers-forster**
This is the first time, i've done a pull request  - what fun :)

### CODE REVIEW

**Comment by chrisroadmap** [comment]
It's only by chance I saw this - GitHub doesn't notify me when people make PRs! Will check this afternoon :)

File: notebooks/Decadaltrends.ipynb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 103,
+   "id": "0c4a24ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in the CSV file with the temperature data\n",
+    "GWI_df = pd.read_csv('https://raw.githubusercontent.com/ClimateIndicator/anthropogenic-warming-assessment/master/global-warming-index/results/Walsh_GMST_timeseries_6048000.csv')\n",
```
Comment (chrisroadmap) [inline notebooks/Decadaltrends.ipynb]: Tristram has changed the folder structure and this path doesn't work now. 

This should: https://raw.githubusercontent.com/ClimateIndicator/anthropogenic-warming-assessment/master/results/Walsh_GMST_timeseries_6048000.csv

The same file is also in the data repo and that will not change if you used the tagged version, but the column headers are different
https://github.com/ClimateIndicator/data/blob/v2023.05.01/data/anthropogenic_warming/Walsh_GMST_timeseries_6048000.csv

**Review by chrisroadmap — CHANGES_REQUESTED** [review]
Hi Piers, very nice 👍 

The path to Tristram's data has changed, see the comment.

The other thing that's nice to do is commit clean notebooks. You can either:
1.  go to Kernel > Restart & Clear Output in jupyter before committing;
2. automatically clean notebooks precommit by activating your conda environment and running `nbstripout --install` from the terminal.

Once those two things are done I will merge. Thanks :)

**Comment by chrisroadmap** [comment]
Oh, one other thing: add to the README under the reproduction instructions - I assume this comes last in the chain.

**Comment by piers-forster** [comment]
After about half an hours effort of trying to work out WTF to do - i think. I've made these changes   - do i need to do anything to try and merge again?

**Review by chrisroadmap — APPROVED** [review]

**Pull request merged**